### ISSUE TITLE
Running *san locally fails trying to run integration tests

### ISSUE THREAD

**Issue by mr-salty** [googleapis/google-cloud-cpp-spanner#600]
Running asan or msan build on my desktop fails:

```
$ ./ci/kokoro/docker/build.sh asan
[...]
INFO: Build completed successfully, 1678 total actions
[...]
INFO: Build completed successfully, 1 total action
================================================================
Running the integration tests Tue Sep 10 17:59:42 UTC 2019
================================================================
/v/ci/kokoro/docker/build-in-docker-bazel.sh: line 79: /c/spanner-integration-tests-config.sh: No such file or directory
```

**Comment by mr-salty** [comment]
I guess this is up to me to fix as operator

**Comment by coryan** [comment]
No, that is unrelated. That link works inside the Docker image.

The issue is that the script tries to run the integration tests, even though the CI configuration files do not exist in your (or my) workstation.

**Comment by mr-salty** [comment]
so should I change it to just skip the integration tests if the config file doesn't exist?  or maybe the default RUN_INTEGRATION_TESTS should be 'no' and we set it explicitly in the ci builds?

**Comment by coryan** [comment]
> so should I change it to just skip the integration tests if the config file doesn't exist?

That is what I would have done (I think).

> or maybe the default RUN_INTEGRATION_TESTS should be 'no' and we set it explicitly in the ci builds?

I think I want to run the integration tests from the command-line, at least some of the time.  And also some of the CI builds do not run the integration tests, so maybe we should not do this.  But maybe I misunderstand.
